examples/CMakeLists.txt

```diff
@@ -46,17 +46,39 @@ IF(DEAL_II_COMPONENT_EXAMPLES)
     PATTERN "DTMB-5415_bulbous_bow.iges"  # step-54
     )
 
-  MESSAGE(STATUS "Setting up examples - Done")
-ENDIF()
+  #
+  # Set up all executables:
+  #
+  FILE(GLOB _steps ${CMAKE_CURRENT_SOURCE_DIR}/step-*/*.cc)
+  FOREACH(_step ${_steps})
+    GET_FILENAME_COMPONENT(_name ${_step} NAME_WE)
+    GET_FILENAME_COMPONENT(_directory ${_step} DIRECTORY)
+    SET(_set_up TRUE)
+    IF(EXISTS "${_directory}/dependency")
+      FILE(READ "${_directory}/dependency" _dependency)
```

Comment:
We already have a subdir `step-XX/doc` where we list meta-info about each step. Maybe stick it in there?

Could we re-use this information in the `step-XX/CMakeLists.txt` files themselves, rather than list things twice?